docs/components/button/README.md

```diff
@@ -395,7 +395,7 @@ Change the button size based on where button is used. Default size is medium. Sm
 <template slot="API">
 <cdr-doc-table-of-contents-shell>
 
-### Properties
+## Props
```

Comment:
why Props over Properties? this seems counter to our naming convention of full rather than shorthand names

Reply:
I put this out a week ago and Maya Ok'd it:

"Can I make an argument for dropping "Properties" in favor of "Props" on the API tab? We only use that word because it comes from vue and their docs say "prop". I think our terms should match the framework to avoid possible confusion."

Reply:
cool I'm fine with that